src/AbstractMetricsManager.h

```diff
@@ -440,6 +440,16 @@ class AbstractMetricsManager
         }
     }
 
+    void shift_bucket(timespec stamp)
```

Comment:
please refactor name to `check_period_shift`

Reply:
done